### PR TITLE
Support device-pixel-ratio in wpt reftests.

### PR DESCRIPTION
**Pull request by Ms2ger**
[Review on Reviewable](https://reviewable.io/reviews/servo/servo/9272)

### CODE REVIEW

**Comment by Ms2ger** [comment]
This is basically #9076.

File: tests/wpt/css-tests/tools/manifest/sourcefile.py [hunk 1]
```diff
@@ -198,6 +198,23 @@ def viewport_size(self):
         return self.viewport_nodes[0].attrib.get("content", None)
 
     @cached_property
+    def dpi_nodes(self):
+        """List of ElementTree Elements corresponding to nodes in a test that
+        specify device pixel ratios"""
+        return self.root.findall(".//{http://www.w3.org/1999/xhtml}meta[@name='device-pixel-ratio']")
```
Comment (andreastt) [inline tests/wpt/css-tests/tools/manifest/sourcefile.py]: Not sure you absolutely need to specify a specific namespace here, but this is fine.

**Comment by andreastt** [comment]
This looks OK to me.  Tests pass locally and the changes to wptrunner and the manifest parser are both reasonable.

**Comment by nox** [comment]
```
Tests with unexpected results:
  ▶ Unexpected subtest result in /html/infrastructure/terminology/plugins/text-plain.html:
  └ PASS [expected FAIL] A user agent must not consider the type text/plain as having a registered plugin.
```

**Pull request merged**